specification/communication/data-plane/Email/preview/2023-15-01-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,413 @@
+      "type": "object",
+      "required": [
+        "email"
+      ],
+      "properties": {
+        "email": {
+          "description": "Email address.",
+          "type": "string",
+          "example": "abc@contoso.com"
+        },
+        "displayName": {
+          "description": "Email display name.",
+          "type": "string",
+          "example": "abc"
+        }
+      }
+    },
+    "EmailContent": {
+      "description": "Content of the email.",
+      "type": "object",
+      "required": [
+        "subject"
+      ],
+      "properties": {
+        "subject": {
+          "description": "Subject of the email message",
+          "type": "string",
+          "example": "An exciting offer especially for you!"
+        },
+        "plainText": {
+          "description": "Plain text version of the email message.",
+          "type": "string",
+          "example": "This exciting offer was created especially for you, our most loyal customer."
+        },
+        "html": {
+          "description": "Html version of the email message.",
+          "type": "string",
+          "example": "<html><head><title>Exciting offer!</title></head><body><h1>This exciting offer was created especially for you, our most loyal customer.</h1></body></html>"
+        }
+      }
+    },
+    "EmailImportance": {
+      "description": "The importance type for the email.",
+      "enum": [
+        "high",
+        "normal",
+        "low"
+      ],
+      "type": "string",
+      "default": "normal",
+      "x-ms-enum": {
+        "name": "EmailImportance",
+        "modelAsString": true
+      }
+    },
+    "EmailAttachment": {
+      "description": "Attachment to the email.",
+      "type": "object",
+      "required": [
+        "name",
+        "type",
+        "contentBytesBase64"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name of the attachment",
+          "type": "string",
+          "example": "attachment.pdf"
+        },
+        "type": {
+          "description": "MIME type of the content being attached.",
+          "type": "string",
+          "example": "text/plain"
+        },
+        "contentBytesBase64": {
+          "description": "Base64 encoded contents of the attachment",
+          "type": "string",
+          "example": "TWFueSBoYW5kcyBtYWtlIGxpZ2h0I"
+        }
+      }
+    },
+    "EmailRecipients": {
+      "description": "Recipients of the email",
+      "type": "object",
+      "required": [
+        "to"
+      ],
+      "properties": {
+        "to": {
+          "description": "Email To recipients",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EmailAddress"
+          }
+        },
+        "CC": {
+          "description": "Email CC recipients",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EmailAddress"
+          }
+        },
+        "bCC": {
```

Comment:
Can we change the casing to completely lower-case `cc` and `bcc`? Guidelines say to use lowerCamelCase but `cC` and `bCC` look like they're asking for developers to make typos.

Reply:
Follow up about this whether we can/should override it per language and keep it all lower case in swagger.

Reply:
Overriding can be done either in the markdown file (if it's specific to a certain language) or in the swagger itself if it gets applied to all languages

Reply:
For now, I've updated it to be all lower case in the swagger. But added an override in swagger to use all uppercase letters. Will update if I hear any further feedback on this from ARB.